examples/tutorials/sst2_classification_non_distributed.py

```diff
@@ -111,10 +116,14 @@
 #
 # ::
 #
+#   def batch_transform(x):
+#       return {"token_ids": text_transform(x["text"]), "target": label_transform(x["label"])}
```

Comment:
Note that `label_transform` was never defined in this file

Reply:
Thanks for catching this. The `SST2` dataset returns an integer label in the range `0-1`. @parmeet do you think we can remove the `label_transform` all-together since we don't need to map a text label to an index?

Reply:
Yes, the target column need no transformation now. Thanks for the catch. We should remove it. 

`"target": label_transform(x["label"] -> "target": x["label"]`